packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -20,10 +20,21 @@ var _ = require('lodash');
 var rhosts = /github|bitbucket|gitlab/i;
 var rtag = /tag:\s*[v=]?(.+?)[,\)]/gi;
 
+function semverTagsPromise(options) {
+  var semverTagsDeferred = Q.defer();
```

Comment:
Also packages like `gitSemverTags` was written long time ago. I think now they could be changed to return a promise.